src/utils/Parsing.fs

```diff
@@ -399,28 +412,28 @@ module internal Implementation =
                           finished <- true;                                                                             
                           valueStack.Push(ValueInfo(res, lhsPos.[0], lhsPos.[1])) 
                     | RecoverableParseError ->
-#if DEBUG
+#if __DEBUG
                           if Flags.debug then Console.WriteLine("RecoverableParseErrorException...\n");
 #endif
                           popStackUntilErrorShifted(None);
                           // User code raised a Parse_error. Don't report errors again until three tokens have been shifted 
                           errorSuppressionCountDown <- 3
                 elif kind = errorFlag then (
-#if DEBUG
+#if __DEBUG
```

Comment:
It's not my code, is fslexyac, i copiare the source
I'll maintaib same behaviour of VisualFSharp, if DEBUG should print i'll define the SHOW_INFO ( orwhaterver) in debug configuration only